src/util-exception-policy.c

```diff
@@ -173,7 +173,7 @@ enum ExceptionPolicy ExceptionPolicyParse(const char *option, const bool support
         } else {
             FatalErrorOnInit(
                     "\"%s\" is not a valid exception policy value. Valid options are drop-flow, "
-                    "pass-flow, bypass, drop-packet, pass-packet or ignore.",
+                    "pass-flow, bypass, reject, drop-packet, pass-packet or ignore.",
```

Comment:
nit: compared to the other changes in this commit, this seems out of place. Can you move it into it's own commit?

Reply:
Removed the change from this commit. Didn't keep it in a separate commit, as this message ended up in a new function, so felt that the update could happen more naturally in that same commit.